test/sdk/go/sdk-client-test.go

```diff
@@ -90,8 +90,11 @@ func main() {
 	if err != nil {
 		log.Fatalf("Could not set annotation: %s", err)
 	}
+	if err = sdk.Reserve(5 * time.Second); err != nil {
```

Comment:
nit: should be `if err := sdk.Reserve(...); ...`

Reply:
I would respectfully disagree. If I do that, `err` becomes a shadow variable of the previously defined `err` variable.

Reply:
My experience with go idioms is to pretty much always scope variables as narrowly as possible, which often causes `err` (since it is a common name) to be shadowed in longer functions. I find this to be perfectly normal, but I can see in a quick google search that other people find it super confusing so I'm ok if you want to leave this as-is.

Reply:
Also my editor complains when I do this :smile: